extensions/positron-r/package.nls.json

```diff
@@ -30,8 +30,8 @@
 	"r.menu.packageCheck.title": "Check R Package",
 	"r.menu.packageDocument.title": "Document R Package",
 	"r.menu.rmarkdownRender.title": "Render Document With R Markdown",
-	"r.configuration.title": "Positron R Language Pack",
-	"r.configuration.title-dev": "Positron R Language Pack (advanced)",
+	"r.configuration.title": "R",
+	"r.configuration.title-dev": "Advanced Settings",
```

Comment:
I am considering changing this title to: `Advanced` since the `Settings` portion is a bit redundant. Let me know if anyone strongly disagrees. The contribution point docs mention avoiding these terms since they are redundant as well: https://code.visualstudio.com/api/references/contribution-points#contributes.configuration

Reply:
Yes, let's _not_ use "Settings" 👍